spec/tapioca/dsl/compilers/kredis_spec.rb

```diff
@@ -0,0 +1,209 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "spec_helper"
+
```

Comment:
Looks like kredis is only available for Rails 7+. We can use the `require` logic above to skip these tests as well in case it isn't loaded.

Reply:
I had forgot to add the gem requirement to `Gemfile-rails-6-1`, Kredis itself requires Active {Model,Support} >= 6.0.